### PR TITLE
[zh-cn]SYNC annotations.md

### PR DESCRIPTION
**Pull request by kinzhi**

### CODE REVIEW

File: content/zh-cn/docs/concepts/overview/working-with-objects/annotations.md [hunk 1]
```diff
@@ -137,11 +137,11 @@ If the prefix is omitted, the annotation Key is presumed to be private to the us
 <!--
 The `kubernetes.io/` and `k8s.io/` prefixes are reserved for Kubernetes core components.
 
-For example, here's the configuration file for a Pod that has the annotation `imageregistry: https://hub.docker.com/` :
+For example, here's a manifest for a Pod that has the annotation `imageregistry: https://hub.docker.com/` :
 -->
 `kubernetes.io/` 和 `k8s.io/` 前缀是为 Kubernetes 核心组件保留的。
 
-例如，下面是一个 Pod 的配置文件，其注解中包含 `imageregistry: https://hub.docker.com/`：
+例如，这是一个带有注释 `imageregistry: https://hub.docker.com/` 的 Pod 的清单：
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/overview/working-with-objects/annotations.md:144]:
```suggestion
例如，下面是一个 Pod 的清单，其注解中包含 `imageregistry: https://hub.docker.com/`：
```
The original style is better. `x 的 y 的 z` is a stiff and machine-translated style.

**Comment by windsonsea** [comment]
Thanks.
/lgtm

**Review by ydFu — APPROVED** [review]
/lgtm

**Pull request merged**